Delta.proto

```diff
@@ -0,0 +1,46 @@
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Network.Protocol.Protobuffs <https://github.com/catalyst-network/protocol-protobuffs>
+*
+* Catalyst.Network.Protocol.Protobuffs is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+* 
+* Catalyst.Network.Protocol.Protobuffs is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+* 
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Network.Protocol.Protobuffs If not, see <https://www.gnu.org/licenses/>.
+*/
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package Catalyst.Protocol.Delta;
+
+/**
+ * Delta
+ *
+ * version:
+ * MerkleRoot:
+ * MerklePoda: proof of delegated authority for active wokers
+ * TimeStamp:
+ * STEntries:
+ * CFEntries:
+ * CBEntries: one per active worker
+ */
+ message Delta {
+	uint32 version = 0 [default = 2]; 
```

Comment:
Can we get an explanation on that default value ?

Reply:
Sorry left over of a copy/paste. Default should be 1 for non-confidential transaction based delta (2 for confidential transaction)

Reply:
i've updated this to remove the default value, but would be good to know what the version field for the delta implies.